GeoFireTests/GeoFireTest.m

```diff
@@ -52,9 +52,6 @@ - (void)testGeoFireSetsLocations
                };
 
             XCTAssertEqualObjects(snapshot.value, expected);
-            XCTAssertEqualObjects([snapshot childSnapshotForPath:@"loc1"].priority, @"7zzzzzzzzz");
```

Comment:
Please instead test `g` rather than `priority`. We'd like to make sure everything still works :)

Reply:
Isn't it already tested on the previous line when comparing the whole value of the snapshot?

Reply:
Derp, yes.